alf/trainers/off_policy_trainer.py

```diff
@@ -60,23 +63,24 @@ def train(root_dir,
 
     Args:
         root_dir (str): directory for saving summary and checkpoints
-        env (TFEnvironment): environment for training
+        env_f (Callable[TFEnvironment]): creates an environment for training
         algorithm (OnPolicyAlgorithm): the training algorithm
+        sync_driver (bool): whether use the synchronous driver or asynchronous driver
```

Comment:
A few more words about the difference between these two. So that user don't need to go deep into the code to understand it.